src/react/features/connector/hooks/useSaveHome.js

```diff
@@ -1,10 +1,10 @@
 import { useEffect } from 'react'
 import { useStore, usePromiseDispatch } from '../../../store'
 
-import { saveRoom } from '../../room/actions'
-
-import { connect } from '../client'
-import { channels } from '../../../../shared/constants'
+// import { saveRoom } from '../../room/actions'
+//
+// import { connect } from '../client'
+// import { channels } from '../../../../shared/constants'
```

Comment:
Why these weren't removed but just commented out?

I suggest removing rather than commenting out.

Reply:
True, the whole useSaveHome is a placeholder. I'll remove it.